manageiq-ui-classic.gemspec

```diff
@@ -20,6 +20,12 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
 
   s.add_dependency "rails", "~> 5.0.0", ">= 5.0.0.1"
+  s.add_dependency "patternfly-sass", "~> 3.15.0"
+  s.add_dependency "lodash-rails", "~>3.10.0"
+  s.add_dependency "angular-ui-bootstrap-rails", "~>0.13.0"
+  s.add_dependency "jquery-hotkeys-rails"
+  s.add_dependency "sass-rails"
+  s.add_dependency "coffee-rails"
```

Comment:
Minor but can you sort these alphabetically?  It makes future diffs and additions simpler.